packages/fxa-content-server/app/scripts/views/base.js

```diff
@@ -1137,6 +1137,25 @@ var BaseView = Backbone.View.extend({
         return result;
       });
   },
+
+  /**
+   * Clear validation tooltips and all visible input values.
+   */
+  clearInput() {
+    const $inputEls = this.$('input');
+
+    $inputEls.each((i, inputEl) => {
+      // Called to clear validation tooltips. issues/5680
+      $(inputEl).change();
+    });
+
+    const formEl = this.$('form  input:not(".hidden")');
```

Comment:
_Tiny_ nit, double space between `form` and `input`.

Reply:
I can't do it. After doing the commit message, the extra space disappear.

Reply:
Do you mean when you try committing the selector as `form input:not(".hidden")` it seems to reformat with a double space between `form` and `input`? That seems strange, we shouldn't have a prettier rule doing that.

Reply:
Oh sorry, my mistake

Reply:
My comment may have been ambiguous, I see could have been more precise with what I was pointing out. Thanks for sticking with this, just saw your latest push and it LGTM, once CI gives the green light I'll r+ and merge.